pkgs/top-level/python-packages.nix

```diff
@@ -5816,7 +5816,9 @@ self: super: with self; {
 
   ifconfig-parser = callPackage ../development/python-modules/ifconfig-parser { };
 
-  ifcopenshell = callPackage ../development/python-modules/ifcopenshell { };
+  ifcopenshell = callPackage ../development/python-modules/ifcopenshell {
+    inherit (pkgs) libxml2;
```

Comment:
I have an error about libxml2 missing if I don't add this line, no ideas why.

Reply:
there is also a python package called libxml2

Reply:
Ah I understand thanks, I do need the native libxml2, not the python one, that's why.